Common/src/Common/Service/Table/Formatter/InternalConversationMessage.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
```

Comment:
Enforcing types with declare strict_types?

Reply:
Probably won't add strict type checks for the moment.

Eventually, yes...just not in this PR